src/Console/Command/DescribeCommand.php

```diff
@@ -265,7 +272,8 @@ private function describeSet(OutputInterface $output, $name)
             throw new DescribeNameNotFoundException($name, 'set');
         }
 
-        $ruleSet = new RuleSet(array($name => true));
+        $ruleSetClass = get_class($this->ruleSet);
+        $ruleSet = new $ruleSetClass(array($name => true));
```

Comment:
I don't fully get this change. You injected `RuleSet` to constructor, and then you recreate it here. it's messy.
and more, it's wrong, as constructor accept any interface, so given class could have private constructor